### ISSUE TITLE
WCAG compliance

### ISSUE THREAD

**Issue by mplusb** [WPChill/modula-lite#689]
For the sake of illustrating the need I have added the three changes I made for our sites below. All of the changes are done in fancybox.js 

1.) Add aria-label to thumbnails around line 5195 
var imageCaption = item.opts.caption.replace(/<p>|<\/p>/igm, ''); list.push( '<a href="javascript:;" role="button" aria-label="Click to show image titled '+ imageCaption +'" tabindex="0" data-index="' + i + '"' + (src && src.length ? ' style="background-image:url(' + src + ')"' : 'class="modula-fancybox-thumbs-missing"') + "></a>" ); 

2.) Add ID to Caption for aria-describedby attribute around line 3029 
.html('<p id="modula-caption-'+ current.index +'" class="modula-fancybox-caption__text">' + caption + '</p>'); 

3.) Add aria-describedby and alt attribute to image around line 1996 
.addClass("modula-fancybox-image") .attr("alt", imgCaption) .attr("aria-describedby", 'modula-caption-' + slide.index) .attr("src", slide.src) .appendTo(slide.$content); 

https://secure.helpscout.net/conversation/2036450420/50580?folderId=1289779
